zcbor/zcbor.py

```diff
@@ -1133,9 +1133,9 @@ def is_uint_disambiguated(self):
 
     def all_children_uint_disambiguated(self):
         values = set(child.uint_val() for child in self.value)
-        bit_sizes = set(child.bit_size() for child in self.value)
-        return (len(values) == len(self.value)) and None not in values \
-            and (len(bit_sizes) == 1) and None not in bit_sizes
+        retval = (len(values) == len(self.value)) and None not in values \
+            and max(values) < 0x80000000 and min(values) >= -0x80000000
```

Comment:
These magic values need to have variables with descriptive names.

Reply:
The value 2147483648 is used in other places in this code. Standardization would be nice at least or do what @hakonfam said and make a "CONST" value used everywhere for this like "MAX_NINT". Maybe even make a task about extracting all the limits into something like type_limits.py

Reply:
Added some variables (e.g. INT32_MAX) and used those instead of magic numbers.